q2_demux/plugin_setup.py

```diff
@@ -160,6 +160,43 @@
         citations['hamady2009']]
 )
 
+plugin.methods.register_function(
+    function=q2_demux.partition_samples_single,
+    inputs={'demux': SampleData[SequencesWithQuality]},
+    parameters={},
+    outputs=[
+        ('partitioned_demux', Collection[SampleData[SequencesWithQuality]]),
+    ],
+    input_descriptions={
+        'demux': 'The demux to partition.'
```

Comment:
```suggestion
        'demux': 'The demultiplexed sequences to partition.'
```
This might be a more clear description for users, and will match with the other method descriptions.